src/react-apps/applications/shared/src/navigation/main-header/appBar.tsx

```diff
@@ -14,8 +14,8 @@ import { menu } from './appBarConfig';
 import ProfileMenu from './profileMenu';
 
 export interface IAppBarComponentProps extends WithStyles<typeof styles> {
-  activeSubHeaderSelection: string;
-  activeLeftMenuSelection: string;
+  activeSubHeaderSelection?: string;
```

Comment:
Why have these become optional? I think you need to have a value for active menu right? if not are you adding any default value later?

Reply:
If showSubHeader=false, you don't have to pass activeSubHeaderSelection prop.